manager/allocator/networkallocator/networkallocator.go

```diff
@@ -282,24 +327,32 @@ func (na *NetworkAllocator) IsTaskAllocated(t *api.Task) bool {
 	}
 
 	// To determine whether the task has its resources allocated,
-	// we just need to look at one network(in case of
+	// we just need to look at one global scope network (in case of
```

Comment:
If there are no networks, this function returns false. If there are only local-scope networks, it returns true. Does that make sense?